src/lsst/cm/tools/cli/commands.py

```diff
@@ -453,19 +453,21 @@ def set_script_status(dbi: DbInterface, status: StatusEnum, script_name: str, **
 @options.sleep_time()
 @options.n_iter()
 @options.verbose()
+@options.log_file()
 def daemon(
     dbi: DbInterface,
     script_method: ScriptMethod,
     max_running: int,
     sleep_time: int,
     n_iter: int,
     verbose: bool,
+    log_file: Any,
```

Comment:
hmm, have we sort of given up on type hints?

Reply:
Could this be "Optional[str]"?

Reply:
Yeah, or `str | None` in even more modern parlance, since we are >= Python 3.10.